### PR TITLE
Javascript question v2 2

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1241,6 +1241,17 @@ const cssQuiz = [
...
+  {
+    Question:
+      "In CSS, When flex-direction is changed to column the main axis becomes the:",
+    Answer: "Cross axis",
+    Distractor1: "X axis",
+    Distractor2: "Y axis",
+    Distractor3: "None of the Above",
+    Explanation:
+      "The main axis is the direction in which flex items are placed. When flex-direction is changed to column the main axis becomes the cross axis.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction"
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1254]: This question was used in another PR you submitted.
Please remove it here

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1990,6 +1990,17 @@ const javascriptQuiz = [
...
+  {
+    Question:
+      "In JavaScript, Can we write a if else statement without paranthesis?",
+    Answer: "Yes, When we have only one line statement",
+    Distractor1: "No ",
+    Distractor2: "Its Impossible",
+    Distractor3: "No I don't Know",
+    Explanation:
+      "Javascript allows us to write a oneline if else statement without paranthesis",
+    Link: "https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript"
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:2003]: Let's go with this rewording of the question and answers

```suggestion
    Question:
      "In JavaScript, which of the following is the correct syntax for a one line if statement?",
    Answer: "if(5>4) console.log("This is true");",
    Distractor1: "if(5>4) <<>> console.log("This is true"); ",
    Distractor2: "if[5>4] console.log("This is true");",
    Distractor3: "((if(5>4) console.log("This is true")));",
    Explanation:
      "In JavaScript, you can create a one line if statement without the curly braces using this syntax: if (condition is true) // short one line of code is executed",
    Link: "https://www.freecodecamp.org/news/javascript-if-else-and-if-then-js-conditional-statements/"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by Shaik-mohd-huzaifa — COMMENTED** [review]
Changes applied

**Pull request closed without merging**